build/make_bundle.sh

```diff
@@ -5,13 +5,17 @@
 set -e
 
 VERSION=`grep "Version.*=.*\".*\"" version/version.go | sed "s,.*Version.*=.*\"\(.*\)\".*,\1,"`
-OLD_VERSIONS="v1beta1 v1alpha3 v1alpha2"
+OLD_VERSIONS="v1 v1beta1 v1alpha3 v1alpha2"
 DOCKER_IO_PATH="docker.io/splunk"
 REDHAT_REGISTRY_PATH="registry.connect.redhat.com/splunk"
 OPERATOR_IMAGE="$DOCKER_IO_PATH/splunk-operator:${VERSION}"
 OLM_CATALOG=deploy/olm-catalog
 OLM_CERTIFIED=deploy/olm-certified
 YAML_SCRIPT_FILE=.yq_script.yaml
+ENTERPRISE_API_LOC=pkg/apis/enterprise
+
+# create a soft link that point to the latest Splunk Enterprise CRDs
+ln -snf `ls -lr $ENTERPRISE_API_LOC| awk 'NF > 2 {print $NF}' | head -1` $ENTERPRISE_API_LOC/latest
```

Comment:
Are you sure this is the right commmand? Because when I run it in my local env. this is what I get - 
```$ ls -lr pkg/apis/enterprise/ | awk 'NF >2 {print$NF}'
v1beta1
v1alpha3
v1alpha2
v1
group.go
```

But if I run `ls -lrt` then I get this - 
```$ ls -lrt pkg/apis/enterprise/ | awk 'NF >2 {print$NF}'
group.go
v1alpha2
v1alpha3
v1beta1
v1
```
So probably better command could be 
`ls -lrt $ENTERPRISE_API_LOC| awk 'NF > 2 {print $NF}' | tail -1`  ?

Reply:
Assuming the versions in chronological order, say, v4, v5, v5.1, it should work. 

Anyhow, I am stashing it for now, as the Golang doesn't like referring the modules through the soft links. Will see how we can achieve it.